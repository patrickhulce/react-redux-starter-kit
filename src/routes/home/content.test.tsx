import * as React from 'react'
import * as Enzyme from 'enzyme'
import * as Adapter from 'enzyme-adapter-react-15'

import Home from './content'

Enzyme.configure({adapter: new Adapter()});

describe('routes/home/container.js', () => {
  it('should render', () => {
    const mockChild = <div className="child" />
    const component = Enzyme.shallow(<Home>{mockChild}</Home>)
    expect(component).toMatchSnapshot()
  })
})
