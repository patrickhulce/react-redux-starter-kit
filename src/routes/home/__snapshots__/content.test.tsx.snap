exports[`routes/home/container.js should render 1`] = `
<div
  className="container">
  <h1>
    <i
      className="fa fa-home" />
     Home
  </h1>
  <Link
    onlyActiveOnIndex={false}
    style={Object {}}
    to="contact">
    Contact
  </Link>
  <div
    className="child" />
</div>
`;
